README.md

```diff
@@ -112,26 +153,9 @@ You can pass in an object as options to the plugin. The following keys are valid
 
 ### config
 
-**IMPORTANT NOTE**
-
-Currently, the plugin accepts a config object like this:
-
-```
-{
-  authRealm: String,
-  authUrl: String,
-  authClientId: String,
-  logoutRedirectUri: String
-}
-```
-
-**This will be deprecated in the next major release.**
```

Comment:
Needs to be a major release then. SHould we implement a more verbose changelog of sorts?

Reply:
We have a fairly verbose release log already